### PR TITLE
update blazorwasm  templates to 3.2.1 of components.webassembly

### PR DESCRIPTION
**Pull request by jennyf19**
#631 

Needs testing for blazorwasm

### CODE REVIEW

File: ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Client/ComponentsWebAssembly-CSharp.Client.csproj [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
-    <PackageReference Include="Microsoft.Authentication.WebAssembly.Msal" Version="3.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="3.2.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Build" Version="3.2.1" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="3.2.1" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.Authentication.WebAssembly.Msal" Version="3.2.1" />
 <!--#if (Hosted) -->
     <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.3" />
 <!--#endif -->
-    <PackageReference Include="System.Net.Http.Json" Version="3.2.0" />
+    <PackageReference Include="System.Net.Http.Json" Version="3.2.1" />
```
Comment (jmprieur) [inline ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Client/ComponentsWebAssembly-CSharp.Client.csproj:17]: I see in the ASP.NET Core templates that they kept ` <PackageReference Include="System.Net.Http.Json" Version="3.2.0" />`
I don't know if that matters

Comment (jennyf19) [reply ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Client/ComponentsWebAssembly-CSharp.Client.csproj:17]: oh i see. i can change it back.

**Comment by jennyf19** [comment]
moving to a draft, as i think i'll need to sync w/javier on this more.

File: ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Client/ComponentsWebAssembly-CSharp.Client.csproj (hunk 1 above)
Comment (jmprieur) [reply ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Client/ComponentsWebAssembly-CSharp.Client.csproj]: This was actually an app registration issue. The redirect URLs were now missing `http://localhost:44365/authentication/login-callback`.

**Comment by jmprieur** [comment]
@jennyf19 
after fixing the app registration, I re tested all the projects generated from the blazorwasm template (hosted or not) and this works fine.

**Review by jmprieur — COMMENTED** [review]
LGTM.
Tested (except the test app as the app as the app reg needs to be updated)

File: ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Client/ComponentsWebAssembly-CSharp.Client.csproj (hunk 1 above)
Comment (jennyf19) [reply ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Client/ComponentsWebAssembly-CSharp.Client.csproj]: yes. it's working now. had to register as "web" platform and not "spa"

**Review by jmprieur — APPROVED** [review]
LGTM. Thanks @jennyf19

**Pull request merged**